specification/app/resource-manager/Microsoft.App/preview/2022-01-01-preview/ContainerApps.json

```diff
@@ -0,0 +1,574 @@
+          },
+          {
+            "name": "name",
+            "in": "path",
+            "description": "Name of the Container App.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "containerAppEnvelope",
+            "in": "body",
+            "required": true,
+            "description": "Properties of a container app that need to be updated",
+            "schema": {
+              "$ref": "#/definitions/ContainerAppPatch"
+            }
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Container App updated successfully.",
+            "schema": {
+              "$ref": "#/definitions/ContainerApp"
+            }
+          },
+          "default": {
+            "description": "Common error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Patch Container App": {
+            "$ref": "./examples/ContainerApps_Patch.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "Configuration": {
+      "description": "Non versioned Container App configuration properties that define the mutable settings of a Container app",
+      "type": "object",
+      "properties": {
+        "secrets": {
+          "description": "Collection of secrets used by a Container app",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Secret"
+          },
+          "x-ms-identifiers": [
+            "name"
+          ]
+        },
+        "activeRevisionsMode": {
+          "description": "ActiveRevisionsMode controls how active revisions are handled for the Container app:\n<list><item>Multiple: multiple revisions can be active. If no value if provided, this is the default</item><item>Single: Only one revision can be active at a time. Revision weights can not be used in this mode</item></list>",
+          "enum": [
+            "multiple",
+            "single"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "ActiveRevisionsMode",
+            "modelAsString": true
+          }
+        },
+        "ingress": {
+          "$ref": "#/definitions/Ingress",
+          "description": "Ingress configurations."
+        },
+        "registries": {
+          "description": "Collection of private container registry credentials for containers used by the Container app",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/RegistryCredentials"
+          },
+          "x-ms-identifiers": [
+            "server"
+          ]
```

Comment:
Had to add this as the Swagger LintDiff step was failing and complaining about it. However, now the SDK azure-resource-manager-schemas fails with this error: 

07:10:18.882 cmdout 	[generateScript.sh] ERROR: Schema violation: Additional properties not allowed: x-ms-identifiers (definitions > Container > properties > env)
07:10:18.883 cmdout 	[generateScript.sh]     - file:///home/vsts/work/1/s/azure-rest-api-specs/specification/app/resource-manager/Microsoft.App/preview/2022-01-01-preview/CommonDefinitions.json:63:4

Please advise on how to fix this.

Reply:
@mentat9 - can you please advise on this as the PR cannot be merged until it is resolved?

Reply:
@mentat9 - can you please help to resolve this validation error?

Reply:
Adding @ericshape and @priyankajayaswal1 in case they have any suggestions.

Reply:
I'm not familiar with this issue. Your reviewer (@chunyu3) is on the SDK team who should be able to help.

Reply:
The 'azure-resource-manager-schema' failure is because the unknown extension 'x-ms-identifiers'. The failure will be resolved by another PR in azure-resource-manager-schema repo. Current for this PR, we can ignore this failure.